pkg/report/linux.go

```diff
@@ -1322,7 +1322,14 @@ var linuxOopses = append([]*oops{
 						compile("(Local variable .* created at:|Uninit was created at:)"),
 						parseStackTrace,
 					},
-					skip: []string{"alloc_skb"},
+					// This is a bit heavy-handed, but in practice KMSAN origin
+					// stacks may contain arbitrary number of generic (like
+					// __alloc_pages, alloc_pages, allocate_slab, slab_alloc,
```

Comment:
We need to add "allocate_slab" to the global skip list ("skipPatterns" above). It relates to the common slab code, we need to skip all slab frames for all bug types.

Reply:
Turns out just adding this frame eliminates the need of KMSAN-specific changes. Thanks!